NuGetPackages/MonoGame.Framework.Content.Pipeline.Build.nuspec

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+    <metadata>
+        <id>MonoGame.Framework.Content.Pipeline.Build</id>
+        <version>0.0.0.0</version>
+        <title>MonoGame.Framework.Content.Pipeline.Build</title>
+        <authors>MonoGameTeam</authors>
+        <owners>MonoGameTeam</owners>
+        <licenseUrl>https://github.com/mono/MonoGame/blob/develop/LICENSE.txt#L2</licenseUrl>
+        <projectUrl>http://monogame.net/</projectUrl>
+        <iconUrl>https://pbs.twimg.com/profile_images/487954549441691649/O3KsHAsb_400x400.png</iconUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>MonoGame is an open source implementation of the Microsoft XNA 4.x Framework.
+      The goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse. We currently support iOS, Android, Windows (both OpenGL and DirectX), Mac OS X, Linux, Windows 8 Store, Windows Phone 8, PlayStation Mobile, and the OUYA console.</description>
+        <summary>MonoGame is an open source cross platform implementation of Microsoft's XNA 4.x Framework</summary>
+        <releaseNotes></releaseNotes>
+        <copyright>Copyright  2015</copyright>
+        <language>en-US</language>
+	</metadata>
+    <files>
+    
+        <!-- NuGet files -->
+        <file src="..\..\NuGetPackages\build\MonoGame.Framework.Content.Pipeline.Build\MonoGame.Framework.Content.Pipeline.Build.targets" target="build\" />
```

Comment:
This .targets sets up the variables for when building content from the NuGet. If the .targets filename matches the package ID and it is in the build folder, it automatically gets added to the project file.